server/src/main/java/org/elasticsearch/action/bulk/TransportBulkAction.java

```diff
@@ -82,24 +84,28 @@
 public class TransportBulkAction extends HandledTransportAction<BulkRequest, BulkResponse> {
 
     private static final Logger logger = LogManager.getLogger(TransportBulkAction.class);
+    public static final String LAZY_ROLLOVER_ORIGIN = "lazy_rollover";
```

Comment:
I think this can be private?

Reply:
It has to be public to be accessible from `org.elasticsearch.xpack.security.authz.AuthorizationUtils`, in this class we pair it with the internal user.